### PR TITLE
[v6r20] Few fixes (one important bug fix)

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*DMS
CHANGE: script for allow/ban SEs now accepting -a/--All switch, for allo status types

*Core
FIX: ClassAdLight: fix to avoid returning a list with empty string

*test
FIX: fully activating RSS in Jenkins tests

ENDRELEASENOTES

### CODE REVIEW

File: Core/Utilities/ClassAd/ClassAdLight.py [hunk 1]
```diff
@@ -114,144 +110,142 @@ def insertAttributeBool( self, name, attribute ):
...
-        return tempString.split( ',' ) 
-    
-    resultList = []    
-    while tempString:      
-      if tempString.find( '{' ) == 0 :
-        end = tempString.find( '}' )
-        resultList.append(tempString[:end+1])
-        tempString = tempString[end+1:]
+        tempString = tempString.replace("\"", "")
+        if tempString:
```
Comment (fstagni) [inline Core/Utilities/ClassAd/ClassAdLight.py]: The only real fix is here. This I believe is a bug. It was manifesting specifically because of this change: https://github.com/DIRACGrid/DIRAC/pull/3875/files?utf8=%E2%9C%93&diff=unified&w=1 and specifically when calling `platformList = classAdJob.getListFromExpression('Platform')` at line 1249 --- the result, for empty platform, was `['']`

File: DataManagementSystem/scripts/dirac-admin-allow-se.py [hunk 2]
```diff
@@ -41,59 +40,64 @@
     check = True
   if switch[0].lower() == "v" or switch[0].lower() == "allowremove":
     remove = True
+  if switch[0].lower() == "a" or switch[0].lower() == "all":
```
Comment (chaen) [inline DataManagementSystem/scripts/dirac-admin-allow-se.py]: `if switch[0].lower() in ( 'a', 'all'):` ?

File: Core/Utilities/ClassAd/ClassAdLight.py (hunk 1 above)
Comment (atsareg) [reply Core/Utilities/ClassAd/ClassAdLight.py]: Yes, this is a bug. I think this can be simpler fixed by returning [] right after the first line in this method:
if not tempString: return []

Comment (fstagni) [reply Core/Utilities/ClassAd/ClassAdLight.py]: Done. Please check latest commit.

File: ResourceStatusSystem/scripts/dirac-rss-set-token.py [hunk 3]
```diff
@@ -76,18 +77,19 @@ def parseSwitches():
...
   switches.setdefault('statusType', None)
   switches.setdefault('releaseToken', False)
+  switches.setdefault('days', 1)
 
   for key in ('element', 'name', 'reason'):
 
     if key not in switches:
       subLogger.error("%s Switch missing" % key)
-      subLogger.error("Please, check documentation below")
+      subLogger.error("Please, check documentation above")
```
Comment (andresailer) [inline ResourceStatusSystem/scripts/dirac-rss-set-token.py]: Why this change? Doesn't the `below` refer to what is printed by `showHelp()`?
